www/src/data/sidebars/doc-links.yaml

```diff
@@ -430,6 +430,8 @@
           link: /docs/image-tutorial/
         - title: Writing Documentation with Docz
           link: /docs/writing-documentation-with-docz/
+        - title: Manipulating Remark ASTs
+          link: /docs/remark-ast/
```

Comment:
I think a better spot for this would be in the Plugin Authoring section. It could build on #13261 and go above or below "Pixabay Source Plugin Tutorial".

Reply:
Sounds good! Depending on when we merge this and if Shannon's PR is merged beforehand, we can then figure out the transition.

As well, Is the name for this tutorial okay or would something like "Remark Plugin Tutorial" or something similar be better?

Reply:
Good point–how about something in between, like "Remark Plugin Tutorial: Manipulating ASTs"?

Reply:
Moved it to the discussed location and simplified the name to just "Remark Plugin Tutorial"